tests/spread/ubuntu-24.04/test-cmd/src/charm.py

```diff
@@ -0,0 +1,40 @@
+#!/usr/bin/env python3
+# Copyright 2025 Alex Lowe
```

Comment:
I think we should avoid specific copyrights when adding files to craft codebase.

Reply:
Oops, good catch! This came straight out of `charmcraft init`.